solutions/swb-reference/SWBv2.postman_collection.json

```diff
@@ -275,17 +302,24 @@
             ],
             "body": {
               "mode": "raw",
-              "raw": "{\n    \"datasetName\": \"testDs\",\n    \"storageName\": \"datasetBucketNameInMainAccount\",\n    \"path\": \"existingFolderName\",\n    \"awsAccountId\": \"mainAccountId\"\n}",
+              "raw": "",
               "options": {
                 "raw": {
                   "language": "json"
                 }
               }
             },
             "url": {
-              "raw": "{{API_URL}}/datasets/import",
+              "raw": "{{API_URL}}/projects/:projectId/datasets/:datasetId/upload-requests",
               "host": ["{{API_URL}}"],
-              "path": ["datasets", "import"]
+              "path": ["projects", ":projectId", "datasets", ":datasetId", "upload-requests"],
+              "query": [
+                {
+                  "key": "filenames",
```

Comment:
i think it'd be better if the example contained the version where filenames is a `string[]` since we support both and the array is more flexible than passing in a single filename per request.